testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -39,5 +39,11 @@ TestNodeProxy : UnitTest {
 		proxy.source = { Silent.ar };
 		this.assertEquals(proxy.isPlaying, false, "Setting the proxy's source should not set isPlaying = true");
 	}
+    
+    test_copy_fadeTimeIsCopied{
+        var oldProxy = proxy.fadeTime_(33);
+        var newProxy = oldProxy.copy;
 
+        this.assertEquals(oldProxy.fadeTime, newProxy.fadeTime);
```

Comment:
now just add a message as third argument to `this.assertEquals`, and it's complete.

Reply:
this assert doesn't need a message; the test name is descriptive enough